.prettierignore

```diff
@@ -0,0 +1 @@
+tokens/
```

Comment:
Not sure about ignoring this folder while executing Prettier. Files contained in the folder are generated but I'm not sure if the intention is to prettify them.

Reply:
I like to exclude it, so when you do the diff the only diff is the generation date 👍